crates/edr_provider/src/requests/serde.rs

```diff
@@ -347,6 +347,63 @@ where
     Ok(result)
 }
 
+/// Value representing timestamp in seconds.
+///
+/// While technically the Yellow Paper allows specifying the timestamp as
+/// any scalar value, using `u64` in practice is more than enough.
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Serialize)]
+pub struct Timestamp(pub u64);
```

Comment:
Let me know if you'd like me to define this in the upper module; maybe it'd make more sense to have it there but we already re-export a type from here so I wasn't sure.